plugin/api.go

```diff
@@ -221,6 +221,11 @@ type API interface {
 	// Minimum server version: 5.6
 	SearchUsers(search *model.UserSearch) ([]*model.User, *model.AppError)
 
+	// SearchPostsInTeam returns a list of posts in a specific team that match the given params.
+	//
+	// Minimum server version: 5.10
+	SearchPostsInTeam(teamId string, paramsList []*model.SearchParams) (*model.PostList, *model.AppError)
```

Comment:
Could you please change the return value to `([]*model.Post, *model.AppError)`? `model.PostList` is more of an internal type.

Reply:
Done.